src/shared/types/index.ts

```diff
@@ -0,0 +1 @@
+export type Player = { name: string; id: string };
```

Comment:
I tried to define a generic player type that can be used in multiple places as the queue, courts, and on the your-turn page, do you think the server can return that?

Reply:
I can return the player like this. Lets move forward